### PR TITLE
feat: improved corner radius

### PR DESCRIPTION
**Pull request by matheusribeirozup**
### Related Issues

Part of #1632 .

### Description and Example

This PR implements new properties for `cornerRadius` for styled widgets and for `ImageView` component.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.
- [x] I have made the documentation changes or I created an issue explaining how to document this change on [Docs issues](https://github.com/ZupIT/beagle-docs/issues). Please link the issue here:

[PR Guide]: https://github.com/ZupIT/beagle/blob/main/doc/contributing/pull_requests.md

### CODE REVIEW

File: android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt [hunk 1]
```diff
@@ -26,6 +26,7 @@ import br.com.zup.beagle.android.components.ImagePath
 import br.com.zup.beagle.android.components.layout.Container
```
Comment (botbeaglezup) [inline android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt:26]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/171ce3c0a88f4b4fd2c1e006d228588fd3e85312/android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt#L26"></span>The import 'br.com.zup.beagle.android.components.layout.Container' is unused.</td>
    </tr>
  </tbody>
</table>

File: android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt [hunk 2]
```diff
@@ -26,6 +26,7 @@ import br.com.zup.beagle.android.components.ImagePath
...
 import br.com.zup.beagle.android.components.layout.Screen
 import br.com.zup.beagle.android.utils.toView
+import br.com.zup.beagle.core.CornerRadius
 import br.com.zup.beagle.core.Style
 import br.com.zup.beagle.ext.applyFlex
```
Comment (botbeaglezup) [inline android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt:31]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/171ce3c0a88f4b4fd2c1e006d228588fd3e85312/android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt#L31"></span>The import 'br.com.zup.beagle.ext.applyFlex' is unused.</td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/171ce3c0a88f4b4fd2c1e006d228588fd3e85312/android/beagle/src/main/java/br/com/zup/beagle/android/components/Image.kt#L29">android/beagle/src/main/java/br/com/zup/beagle/android/components/Image.kt#L29</a> - The import 'br.com.zup.beagle.android.utils.dp' is unused.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/171ce3c0a88f4b4fd2c1e006d228588fd3e85312/android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt#L34">android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt#L34</a> - The import 'br.com.zup.beagle.widget.core.Flex' is unused.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/171ce3c0a88f4b4fd2c1e006d228588fd3e85312/android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt#L35">android/sample/src/main/java/br/com/zup/beagle/sample/fragment/ImageViewFragment.kt#L35</a> - The import 'br.com.zup.beagle.widget.core.FlexDirection' is unused.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by paulomeurerzup — APPROVED** [review]

**Review by uziasferreirazup — APPROVED** [review]

**Pull request merged**